packages/dropdowns/src/Fields/Field.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { createContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import { StyledField } from '../styled';
+import useDropdownContext from '../utils/useDropdownContext';
+
+export const FieldContext = createContext(undefined);
+
+/**
+ * Accepts all `<div>` props
+ */
+function Field(props) {
```

Comment:
Should we use only the arrow or the function notation for consistency?

Reply:
I was mixed with this in a few components, but I'd like to have function notation for functional components.  Even thought we use them interchangeably within classes they do perform differently.

Within the React docs it seems to follow the `function notation` whenever you don't have to worry about scoping. https://reactjs.org/docs/components-and-props.html#function-and-class-components